injector/node_failure.go

```diff
@@ -50,6 +51,10 @@ func NewNodeFailureInjector(spec v1beta1.NodeFailureSpec, config NodeFailureInje
 		return nil, fmt.Errorf("environment variable %s doesn't exist", env.InjectorMountSysrqTrigger)
 	}
 
+	if config.WaitBeforeShutdown == 0 {
```

Comment:
Improve testing by avoiding waiting 10s for nothing, keeping default value to 10s like before